.env.dev

```diff
@@ -1,4 +1,4 @@
-APP_ENV=prod
+APP_ENV=test
```

Comment:
Its a right old mess! You are absolutely right. In a successive PR I re-set it to dev. I'm opting to leave it like this for now. But promise you it will be rectified in a jiffy